test/extended/extended_test.go

```diff
@@ -11,15 +11,28 @@ import (
 	_ "github.com/openshift/origin/test/extended/jobs"
 	_ "github.com/openshift/origin/test/extended/router"
 	_ "github.com/openshift/origin/test/extended/security"
+	e2e "k8s.io/kubernetes/test/e2e"
+	"os"
+	"testing"
 
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
 // init initialize the extended testing suite.
 func init() {
-	exutil.InitTest()
+	// Kubernetes: Pure end to end tests ~ These flags pass through.
+	if os.Getenv("RUN_KUBE_E2E_SUBSET") == "true" {
+		e2e.RegisterFlags()
```

Comment:
- we all agreed in a separate issue that we want an e2e binary of the kube e2es to ship alongside.
- the only issue as jordan suggests: Do we want a separate binary?  I think clayton said 'no' but we can wait for his response...